docs/resources/service-providers.md

```diff
@@ -56,25 +168,26 @@ gaiad config node <host>:<port
 // example: gaiad config node https://77.87.106.33:26657
 ```
 
-If you run your own full-node, just use `tcp://localhost:26657` as the address. 
+If you run your own Full Node locally, use `tcp://localhost:26657` as the address. 
 
-Then, let us set the default value of the `--trust-node` flag:
+Set the default value of the `--trust-node` flag:
 
 ```bash
 gaiad config trust-node false
 
-// Set to true if you run a light-client node, false otherwise
+// Set to true if you run a light-client node
 ```
 
-Finally, let us set the `chain-id` of the blockchain we want to interact with:
+Finally, set the `chain-id` of the blockchain you want to interact with:
 
 ```bash
-gaiad config chain-id cosmoshub-2
+gaiad config chain-id cosmoshub-4
 ```
 
-Next you will find a few useful CLI commands to interact with the Full-Node.
+Next you will learn useful CLI commands to interact with the Full Node.
+You can run these commands as remote control or when you are running it on your local machine.
 
-#### Creating a key-pair
+### Create a Key pair
 
 To generate a new key (default secp256k1 elliptic curve):
```

Comment:
```suggestion
The default key is `secp256k1 elliptic curve`. Use the `gaiad keys` command to list the keys and generate a new key.


```

Reply:
oops not sure what the next content is starting with line 201, we are listing TYPE, ADDRESS, etc... are these parameters for the gaiad keys command?

Reply:
No, that is what the response has as data.